spring-integration-core/src/main/resources/META-INF/spring.integration.default.properties

```diff
@@ -4,3 +4,4 @@ spring.integration.channels.maxBroadcastSubscribers=0x7fffffff
 spring.integration.taskScheduler.poolSize=10
 spring.integration.messagingTemplate.throwExceptionOnLateReply=false
 spring.integration.messagingAnnotations.require.componentAnnotation=false
+spring.integration.messagingGateway.convertReceiveMessage=false
```

Comment:
Disagreed. 
I don't like to encourage end-users to rely on bug.
With the `false` we signal them that there is something wrong and has been fixed.
But if you don't like it, it's your responsibility to switch to `true`.

Otherwise they may think that we have many features which are just result of the bug :smile: